README.md

```diff
@@ -18,7 +18,7 @@ TRANSLATION_SERVICE_ACCOUNT_KEY="..."
 FACEBOOK_AUTH_TOKEN="..."
 
 FORTIS_FEATURE_SERVICE_HOST="..."
-FORTIS_SERVICES_APPINSIGHTS_KEY="..."
+APPINSIGHTS_INSTRUMENTATIONKEY="..."
```

Comment:
I noticed that in `project-fortis-spark` the environment variable is `APPLICATION_INSIGHTS_IKEY`. Do we have a PR for that project as well? Or should we just make this project use that key?